Search/list.json

```diff
@@ -35,47 +35,47 @@
       "default": {
         "searchDefault": "Google",
         "visibleDefaultEngines": [
-          "google", "be.wikipedia.org", "bing", "tut.by", "yahoo", "yandex.by"
+          "google", "be.wikipedia.org", "bing", "tut.by", "yandex.by"
         ]
       },
       "BY": {
         "searchDefault": "Яндекс",
         "visibleDefaultEngines": [
-           "yandex.by", "google-nocodes", "be.wikipedia.org", "bing", "tut.by", "yahoo"
+           "yandex.by", "google-nocodes", "be.wikipedia.org", "bing", "tut.by"
         ]
       },
       "KZ": {
         "searchDefault": "Яндекс",
         "visibleDefaultEngines": [
-           "yandex.by", "google-nocodes", "be.wikipedia.org", "bing", "tut.by", "yahoo"
+           "yandex.by", "google-nocodes", "be.wikipedia.org", "bing", "tut.by"
         ]
       },
       "RU": {
         "searchDefault": "Яндекс",
         "visibleDefaultEngines": [
-           "yandex.by", "google-nocodes", "be.wikipedia.org", "bing", "tut.by", "yahoo"
+           "yandex.by", "google-nocodes", "be.wikipedia.org", "bing", "tut.by"
         ]
       },
       "TR": {
         "searchDefault": "Яндекс",
         "visibleDefaultEngines": [
-           "yandex.by", "google-nocodes", "be.wikipedia.org", "bing", "tut.by", "yahoo"
+           "yandex.by", "google-nocodes", "be.wikipedia.org", "bing", "tut.by"
         ]
       }
     },
     "bn-IN": {
       "default": {
         "searchDefault": "Google",
         "visibleDefaultEngines": [
-          "google", "yahoo-in", "bing", "amazondotcom", "rediff", "wikipedia-bn"
+          "google", "bing", "amazondotcom", "rediff", "wikipedia-bn"
         ]
       }
     },
     "br": {
       "default": {
         "searchDefault": "Google",
         "visibleDefaultEngines": [
-          "google", "yahoo-france", "bing", "twitter", "wikipedia-br"
+          "google", region"bing", "twitter", "wikipedia-br"
```

Comment:
not valid JSON :-)

Reply:
That's what I get for trusting find replace :).